apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/service/CompletionServiceDefault.java

```diff
@@ -223,8 +223,8 @@ public void rejectOutdatedOnboardings(Onboarding onboarding) {
         LocalDateTime now = LocalDateTime.now();
         onboardingRepository
                 .update("status = ?1 and updatedAt = ?2 ", REJECTED, now)
-                .where("productId = ?1 and institution.origin = ?2 and institution.originId = ?3 and status = PENDING or status = TOBEVALIDATED",
-                        onboarding.getProductId(), onboarding.getInstitution().getOrigin(), onboarding.getInstitution().getOriginId());
+                .where("productId = ?1 and institution.origin = ?2 and institution.originId = ?3 and _id != ?4 and status = PENDING or status = TOBEVALIDATED",
```

Comment:
This code isn't covered by test, could you implement additional testing methods?